### ISSUE TITLE
Object.addLeftTile is deprecated.

### ISSUE THREAD

**Issue by blomma** [lee-dohm/soft-wrap-indicator#9]
Use version ^1.0.0 of the status-bar Service API.
```
Object.addLeftTile (/Users/blomma/Applications/Atom.app/Contents/Resources/app/node_modules/status-bar/lib/main.js:172:16)
soft-wrap-indicator.attach (/Users/blomma/.atom/packages/soft-wrap-indicator/lib/soft-wrap-indicator-view.coffee:19:23)
```

**Comment by lee-dohm** [comment]
Thanks for reporting!

**Comment by lee-dohm** [comment]
Fixed in v0.6.1
